docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx

```diff
@@ -39,7 +39,7 @@ In this tutorial, you build a smart contract with a piece of private state that
 
 ## Prerequisites
 
-This tutorial has been tested with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.9.0` and [SnarkyJS](https://www.npmjs.com/package/snarkyjs) `0.11.0`.
+This tutorial has been tested with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.13.0` and [o1js](https://www.npmjs.com/package/o1js) `0.12.1`.
```

Comment:
if you have tested with 12.1, great! We have not officially tested with o1js, so the SnarkyJS release 11.0 is accurate